spec/lib/object_builders/qhp_rate_builder_spec.rb

```diff
@@ -3,7 +3,7 @@
 
 describe QhpRateBuilder do
 
-  let(:plan){ FactoryGirl.create(:plan) }
```

Comment:
Why does this plan need to be locked by year?

Reply:
are we going to need a qhp_rate_builder for 2018 and associated test?

Reply:
we already had code for 2018 qhp_rate_builder, we only need associated test